docs/installation/zoom-configuration/README.md

```diff
@@ -6,21 +6,21 @@ description: >-
 
 # Zoom Configuration
 
-Zoom version 1.5 supports one authentication method for users to connect Mattermost and Zoom: OAuth 
-
-* There are two types of OAuth Zoom Apps you can create.  You can use either one with this Zoom plugin depending on your organization's security and UX preferences.  \(**Account** or **User** Level Apps\)
-  * **Account-Level App**
-    * Users do not need to authorize their Zoom accounts before starting their first Zoom meeting.  The only requirement is that their Mattermost account uses the same email address as their Zoom account. 
-    * Users cannot connect their Mattermost/Zoom accounts if their emails do not match.
-  * **User Level App**
-    * Each user will need to connect their Zoom account with their Mattermost account before they can use the integration. When they try to create a meeting for the first time, they'll receive a message to connect their account, and will need to click **Approve** on the pop-up confirmation notice.
+Zoom version 1.5 supports one authentication method for users to connect Mattermost and Zoom: OAuth.
+
+* There are two types of OAuth Zoom apps you can create - **Account** or **User** level apps. You can use either one with this Zoom plugin depending on your organization's security and UX preferences.
+
+  * **Account-level app**
+    * Users don't need to authorize their Zoom accounts before starting their first Zoom meeting. The only requirement is that their Mattermost account uses the same email address as their Zoom account.
+    * Users can't connect their Mattermost/Zoom accounts if their emails do not match.
+  * **User-level app**
+    * Each user needs to connect their Zoom account with their Mattermost account before they can use the integration. When they try to create a meeting for the first time, they'll receive a message to connect their account, and will need to click **Approve** on the pop-up confirmation notice.
```

Comment:
Not sure if worth it, but the actual flow is:
- They try to create a meeting
- They must follow a link
- The link will redirect to a zoom owned website
- On this site, if they are not logged in, they will be asked to log in into zoom
- Once they are logged in, they will see the "approve" button.

Probably with what is already written, everything is clear, but wanted to make sure.